config/imsim-config.yaml

```diff
@@ -230,6 +229,8 @@ output:
     nproc: 1    # Change this to work on multiple CCDs at once.
     nfiles: 1   # Default is all 189 CCDs.  Set to 1 while testing.
 
+    camera_class: LsstCam
```

Comment:
I would prefer this to be just `camera`, rather than `camera_class`, but I'll defer to others' opinions if people disagree.  From a user's perspective, it feels like they should be identifying what camera they want to simulate, not some python class in the lsst stack they want to use.  

In fact, if we start having people use DECam or other non Rubin camera's, I'd want us to special case them and prepend the right modules, not make them know to write `lsst.obs.decam.DarkEnergyCamera`.  Rather, I'd want us to list somewhere the values that we support and internally prepend whatever the right thing is in terms of the lsst naming details.

Reply:
`camera` is fine as far as I'm concerned.  In principle, people can write new obs packages or add cameras to existing obs packages, so asking users to give the fully qualified class for these cases doesn't seem like much of an ask, but I guess we can just support the obs.lsst cameras for now.

Reply:
Not having the users need to know about the camera class names (or that there are different classes) also feels correct to me.

I think there will be a group of people using the program unfamiliar with DM who will just want to choose from a list of possible camera names.